### DISCUSSION TITLE
Nuxt 3 Deployment on Azure Static Web App: first request feels like a cold start (20-30secs loading time)

### DISCUSSION THREAD

**Discussion by lukas-leonhardt** [nuxt/nuxt#30250] [Questions]
I deployed my nuxt application with the nitro azure preset to an azure static web app and the first loading of the app after some time without a request (5 minutes) takes 20-30 seconds. After that it is really fast. I already deployed a hello world nuxt application with the same issue. Some people already recommended that I could ping the app every minute or so, but that is for me only a temporal solution. Does anyone already faced the same issue?

**Comment by jwhazel** [comment]
We've had numerous issues with SWA in production, one of which being cold starts. Trying to get any type of support out of them for this product is a nightmare. I don't have any great advice other than to switch to a competitor who offers static site with edge functions like Vercel or Netlify. 

If you *have* to live in the Azure realm, then it makes more sense currently to run under SSR with a Web App/App Service Plan funneled into Front Door.
